src/components/paginator-render/paginator-render.tsx

```diff
@@ -0,0 +1,668 @@
+            if ((this.model as PaginatorRenderNumericModel)?.urlMapping) {
+              return (
+                <li>
+                  <a
+                    key={index}
+                    id={page.toString()}
+                    part={tokenMap({
+                      [PAGINATOR_PARTS_DICTIONARY.PAGE_ACTIVE]:
+                        this.actualPage === page
+                    })}
+                    href={(
+                      this.model as PaginatorRenderNumericModel
+                    ).urlMapping(page)}
+                  >
+                    {page}
+                  </a>
+                </li>
+              );
+            }
+            // Else, return a button with page number as value
+            return (
+              <li>
+                <button
+                  key={index}
+                  part={tokenMap({
+                    [PAGINATOR_PARTS_DICTIONARY.PAGE_ACTIVE]:
+                      this.actualPage === page
+                  })}
+                  value={page}
+                >
+                  {page}
+                </button>
+              </li>
+            );
+          }
+          // if page is "-1", it renders a span with the ellipsis
+          return (
+            <li>
+              <span
+                key={index}
+                part={tokenMap({
+                  [PAGINATOR_PARTS_DICTIONARY.ELLIPSIS]: true
+                })}
+              >
+                {this.translations.text.ellipsis}
+              </span>
+            </li>
+          );
+        })}
+      </ol>
+    );
+  };
+
+  #renderNavigationControlsInfo = () => (
+    <div part={tokenMap({ [PAGINATOR_PARTS_DICTIONARY.INFO]: true })}>
+      <span part={tokenMap({ [PAGINATOR_PARTS_DICTIONARY.INFO__TEXT]: true })}>
+        {this.translations.text.showingPage
+          .replace(ACTUAL_PAGE, this.actualPage?.toString())
+          .replace(
+            TOTAL_PAGES,
+            (this.totalPages > 0 && this.totalPages?.toString()) ||
+              this.translations.text.unknownPages
+          )}
+      </span>
+    </div>
+  );
+
+  #renderNavigationGoTo = () => (
+    <div part={tokenMap({ [PAGINATOR_PARTS_DICTIONARY.GO_TO]: true })}>
+      {this.translations.text.page}
+      <input
+        part={tokenMap({ [PAGINATOR_PARTS_DICTIONARY.GO_TO__INPUT]: true })}
+        type="number"
+        min="1"
+        max={this.totalPages}
+        value={this.actualPage}
+        onKeyDown={this.#handleGoToOnKeyDown}
+        onBlur={this.#handleGoToOnBlur}
+      />
+      {this.translations.text.of}
+      {this.translations.text.of &&
+        ((this.totalPages > 0 && this.totalPages?.toString()) ||
+          this.translations.text.unknownPages)}
+    </div>
+  );
+
+  render() {
+    // Array with controls for mapping
+    const controls = [
+      {
+        name: "itemsPerPage",
+        render: () => this.#renderItemsPerPage(),
+        show: this.showItemsPerPage
```

Comment:
I think we can only compute this array in the `connectedCallback` and in the `Watch("order")` if the make the `show` property work as a function that is evaluated on render.

```suggestion
        show: () => this.showItemsPerPage
```